Build failure: cbfmt

### Steps To Reproduce

Steps to reproduce the behavior:
1. build `cbfmt`

### Build log

```
@nix { "action": "setPhase", "phase": "unpackPhase" }
Running phase: unpackPhase
unpacking source archive /nix/store/crh5zm2xxxz1plpyr18sfbiwd0297aax-source
source root is source
Executing cargoSetupPostUnpackHook
unpacking source archive /nix/store/ffhm3x5dfki8p0v8cpygbc6hvxv8vcp9-cbfmt-0.2.0-vendor.tar.gz
Finished cargoSetupPostUnpackHook
@nix { "action": "setPhase", "phase": "patchPhase" }
Running phase: patchPhase
Executing cargoSetupPostPatchHook
Validating consistency between /private/tmp/nix-build-cbfmt-0.2.0.drv-0/source/Cargo.lock and /private/tmp/nix-build-cbfmt-0.2.0.drv-0/cbfmt-0.2.0-vendor.tar.gz/Cargo.lock
Finished cargoSetupPostPatchHook
@nix { "action": "setPhase", "phase": "updateAutotoolsGnuConfigScriptsPhase" }
Running phase: updateAutotoolsGnuConfigScriptsPhase
@nix { "action": "setPhase", "phase": "configurePhase" }
Running phase: configurePhase
@nix { "action": "setPhase", "phase": "buildPhase" }
Running phase: buildPhase
Executing cargoBuildHook
++ env CC_AARCH64_APPLE_DARWIN=/nix/store/vwh2qqal1q101wv4jy40giavp73q0msd-clang-wrapper-16.0.6/bin/cc CXX_AARCH64_APPLE_DARWIN=/nix/store/vwh2qqal1q101wv4jy40giavp73q0msd-clang-wrapper-16.0.6/bin/c++ CARGO_TARGET_AARCH64_APPLE_DARWIN_LINKER=/nix/store/vwh2qqal1q101wv4jy40giavp73q0msd-clang-wrapper-16.0.6/bin/cc CC_AARCH64_APPLE_DARWIN=/nix/store/vwh2qqal1q101wv4jy40giavp73q0msd-clang-wrapper-16.0.6/bin/cc CXX_AARCH64_APPLE_DARWIN=/nix/store/vwh2qqal1q101wv4jy40giavp73q0msd-clang-wrapper-16.0.6/bin/c++ CARGO_TARGET_AARCH64_APPLE_DARWIN_LINKER=/nix/store/vwh2qqal1q101wv4jy40giavp73q0msd-clang-wrapper-16.0.6/bin/cc CARGO_BUILD_TARGET=aarch64-apple-darwin HOST_CC=/nix/store/vwh2qqal1q101wv4jy40giavp73q0msd-clang-wrapper-16.0.6/bin/cc HOST_CXX=/nix/store/vwh2qqal1q101wv4jy40giavp73q0msd-clang-wrapper-16.0.6/bin/c++ cargo build -j 10 --target aarch64-apple-darwin --frozen --profile release
[1m[32m   Compiling[0m memchr v2.5.0
[1m[32m   Compiling[0m proc-macro2 v1.0.43
[1m[32m   Compiling[0m unicode-ident v1.0.3
[1m[32m   Compiling[0m quote v1.0.21
[1m[32m   Compiling[0m autocfg v1.1.0
[1m[32m   Compiling[0m syn v1.0.99
[1m[32m   Compiling[0m cc v1.0.73
[1m[32m   Compiling[0m regex-syntax v0.6.27
[1m[32m   Compiling[0m libc v0.2.131
[1m[32m   Compiling[0m futures-core v0.3.23
[1m[32m   Compiling[0m slab v0.4.7
[1m[32m   Compiling[0m futures-task v0.3.23
[1m[32m   Compiling[0m futures-channel v0.3.23
[1m[32m   Compiling[0m aho-corasick v0.7.18
[1m[32m   Compiling[0m tree-sitter v0.20.8
[1m[32m   Compiling[0m regex v1.6.0
[1m[32m   Compiling[0m once_cell v1.13.0
[1m[32m   Compiling[0m futures-sink v0.3.23
[1m[32m   Compiling[0m pin-project-lite v0.2.9
[1m[32m   Compiling[0m futures-util v0.3.23
[1m[32m   Compiling[0m cfg-if v1.0.0
[1m[32m   Compiling[0m serde_derive v1.0.143
[1m[32m   Compiling[0m log v0.4.17
[1m[32m   Compiling[0m indexmap v1.9.1
[1m[32m   Compiling[0m serde v1.0.143
[1m[32m   Compiling[0m pin-utils v0.1.0
[1m[32m   Compiling[0m crossbeam-utils v0.8.11
[1m[32m   Compiling[0m futures-io v0.3.23
[1m[32m   Compiling[0m tree-sitter-md v0.1.1
[1m[32m   Compiling[0m tree-sitter-rst v0.1.0
[1m[32m   Compiling[0m tree-sitter-org v1.3.0
[1m[32m   Compiling[0m bstr v0.2.17
[1m[32m   Compiling[0m tokio v1.20.1
[1m[32m   Compiling[0m unicode-width v0.1.9
[1m[32m   Compiling[0m same-file v1.0.6
[1m[32m   Compiling[0m os_str_bytes v6.3.0
[1m[32m   Compiling[0m unicode-linebreak v0.1.2
[1m[32m   Compiling[0m hashbrown v0.12.3
[1m[32m   Compiling[0m smawk v0.3.1
[1m[32m   Compiling[0m fnv v1.0.7
[1m[32m   Compiling[0m clap_lex v0.2.4
[1m[32m   Compiling[0m walkdir v2.3.2
[1m[32m   Compiling[0m globset v0.4.9
[1m[32m   Compiling[0m futures-macro v0.3.23
[1m[32m   Compiling[0m textwrap v0.15.0
[1m[32m   Compiling[0m tokio-macros v1.8.0
[1m[32m   Compiling[0m thiserror-impl v1.0.32
[1m[32m   Compiling[0m thread_local v1.1.4
[1m[32m   Compiling[0m num_cpus v1.13.1
[1m[32m   Compiling[0m atty v0.2.14
[1m[32m   Compiling[0m bitflags v1.3.2
[1m[32m   Compiling[0m strsim v0.10.0
[1m[32m   Compiling[0m lazy_static v1.4.0
[1m[32m   Compiling[0m termcolor v1.1.3
[1m[32m   Compiling[0m clap v3.2.17
[1m[32m   Compiling[0m ignore v0.4.18
[1m[32m   Compiling[0m thiserror v1.0.32
[1m[32m   Compiling[0m futures-executor v0.3.23
[1m[32m   Compiling[0m futures v0.3.23
[1m[32m   Compiling[0m toml v0.5.9
[1m[32m   Compiling[0m cbfmt v0.2.0 (/private/tmp/nix-build-cbfmt-0.2.0.drv-0/source)
[0m[1m[38;5;9merror[0m[0m[1m: linking with `/nix/store/vwh2qqal1q101wv4jy40giavp73q0msd-clang-wrapper-16.0.6/bin/cc` failed: exit status: 1[0m
[0m  [0m[0m[1m[38;5;12m|[0m
[0m  [0m[0m[1m[38;5;12m= [0m[0m[1mnote[0m[0m: env -u IPHONEOS_DEPLOYMENT_TARGET -u TVOS_DEPLOYMENT_TARGET LC_ALL="C" PATH="/nix/store/y24qqyfbnnxza5841hd9hnda5icmzymq-rustc-1.74.0/lib/rustlib/aarch64-apple-darwin/bin:/nix/store/30m4ghksg1dlnwfy3qybfixjwnrhgrh9-cargo-1.74.0/bin:/nix/store/6a8n27y9r10pnr653344yybxr4qpd7g9-cargo-auditable-0.6.1/bin:/nix/store/rp517rlw5kbxcwm382ap5lg0jvn4nakr-auditable-cargo-1.74.0/bin:/nix/store/30m4ghksg1dlnwfy3qybfixjwnrhgrh9-cargo-1.74.0/bin:/nix/store/i2q9ff0fpgr20zza6kr6nlpki6mn43if-rustc-wrapper-1.74.0/bin:/nix/store/vwh2qqal1q101wv4jy40giavp73q0msd-clang-wrapper-16.0.6/bin:/nix/store/3vzbr8yb4c233lhi1qz78b8kaaxrjqcj-clang-16.0.6/bin:/nix/store/pzajj7gw0nazs3vzp6jld53q8ird5izr-coreutils-9.4/bin:/nix/store/0i7w6qsij8akg6h39qn67w9ig5hirv1a-cctools-binutils-darwin-wrapper-16.0.6-973.0.1/bin:/nix/store/i4sfjfz08vmnrj84asl2j5imakga12za-cctools-binutils-darwin-16.0.6-973.0.1/bin:/nix/store/pzajj7gw0nazs3vzp6jld53q8ird5izr-coreutils-9.4/bin:/nix/store/6x5inimwy8rlvf678yy080zccbw71alb-findutils-4.9.0/bin:/nix/store/01bzxsyn8dqz9fsdhp9r6ya796g5ydmf-diffutils-3.10/bin:/nix/store/9z9k8dwhhiryr7z2pz70zca0vwa15yqc-gnused-4.9/bin:/nix/store/zsm4xd527427rnjfnvfhvsld2n57b822-gnugrep-3.11/bin:/nix/store/hhc2x65zvp89lh15grqk21r6dcvifggy-gawk-5.2.2/bin:/nix/store/gclclfhq9blxp32lv8kxm9iv5060rx5f-gnutar-1.35/bin:/nix/store/crpi4clk07gy760ydbb710f74dy8q6c9-gzip-1.13/bin:/nix/store/qay0pa86lb712162d6m5lyw62iczb71d-bzip2-1.0.8-bin/bin:/nix/store/zc9qsb0rzlw0ckhy7097frcwb4pkfx3d-gnumake-4.4.1/bin:/nix/store/6nxav88iiz0g8m598xy643f8hhdz5kkx-bash-5.2-p21/bin:/nix/store/kizmwlnj40d11ixbap74c0syaqxn05wc-patch-2.7.6/bin:/nix/store/xcy0fr1dangwh9r9isay6fqbg91g5b3h-xz-5.4.5-bin/bin:/nix/store/x1bzwj6pz7f3ajg5y36x07b1328jzx7r-file-5.45/bin" VSLANG="1033" ZERO_AR_DATE="1" "/nix/store/vwh2qqal1q101wv4jy40giavp73q0msd-clang-wrapper-16.0.6/bin/cc" "-arch" "arm64" "/private/tmp/nix-build-cbfmt-0.2.0.drv-0/rustcUB2Kwc/symbols.o" "/private/tmp/nix-build-cbfmt-0.2.0.drv-0/source/target/aarch64-apple-darwin/release/deps/cbfmt-56cc8c904f3db47c.cbfmt.2f45c14f779ed0a4-cgu.00.rcgu.o" "/private/tmp/nix-build-cbfmt-0.2.0.drv-0/source/target/aarch64-apple-darwin/release/deps/cbfmt-56cc8c904f3db47c.cbfmt.2f45c14f779ed0a4-cgu.01.rcgu.o" "/private/tmp/nix-build-cbfmt-0.2.0.drv-0/source/target/aarch64-apple-darwin/release/deps/cbfmt-56cc8c904f3db47c.cbfmt.2f45c14f779ed0a4-cgu.02.rcgu.o" "/private/tmp/nix-build-cbfmt-0.2.0.drv-0/source/target/aarch64-apple-darwin/release/deps/cbfmt-56cc8c904f3db47c.cbfmt.2f45c14f779ed0a4-cgu.03.rcgu.o" "/private/tmp/nix-build-cbfmt-0.2.0.drv-0/source/target/aarch64-apple-darwin/release/deps/cbfmt-56cc8c904f3db47c.cbfmt.2f45c14f779ed0a4-cgu.04.rcgu.o" "/private/tmp/nix-build-cbfmt-0.2.0.drv-0/source/target/aarch64-apple-darwin/release/deps/cbfmt-56cc8c904f3db47c.cbfmt.2f45c14f779ed0a4-cgu.05.rcgu.o" "/private/tmp/nix-build-cbfmt-0.2.0.drv-0/source/target/aarch64-apple-darwin/release/deps/cbfmt-56cc8c904f3db47c.cbfmt.2f45c14f779ed0a4-cgu.06.rcgu.o" "/private/tmp/nix-build-cbfmt-0.2.0.drv-0/source/target/aarch64-apple-darwin/release/deps/cbfmt-56cc8c904f3db47c.cbfmt.2f45c14f779ed0a4-cgu.07.rcgu.o" "/private/tmp/nix-build-cbfmt-0.2.0.drv-0/source/target/aarch64-apple-darwin/release/deps/cbfmt-56cc8c904f3db47c.cbfmt.2f45c14f779ed0a4-cgu.08.rcgu.o" "/private/tmp/nix-build-cbfmt-0.2.0.drv-0/source/target/aarch64-apple-darwin/release/deps/cbfmt-56cc8c904f3db47c.cbfmt.2f45c14f779ed0a4-cgu.09.rcgu.o" "/private/tmp/nix-build-cbfmt-0.2.0.drv-0/source/target/aarch64-apple-darwin/release/deps/cbfmt-56cc8c904f3db47c.cbfmt.2f45c14f779ed0a4-cgu.10.rcgu.o" "/private/tmp/nix-build-cbfmt-0.2.0.drv-0/source/target/aarch64-apple-darwin/release/deps/cbfmt-56cc8c904f3db47c.cbfmt.2f45c14f779ed0a4-cgu.11.rcgu.o" "/private/tmp/nix-build-cbfmt-0.2.0.drv-0/source/target/aarch64-apple-darwin/release/deps/cbfmt-56cc8c904f3db47c.cbfmt.2f45c14f779ed0a4-cgu.12.rcgu.o" "/private/tmp/nix-build-cbfmt-0.2.0.drv-0/source/target/aarch64-apple-darwin/release/deps/cbfmt-56cc8c904f3db47c.cbfmt.2f45c14f779ed0a4-cgu.13.rcgu.o" "/private/tmp/nix-build-cbfmt-0.2.0.drv-0/source/target/aarch64-apple-darwin/release/deps/cbfmt-56cc8c904f3db47c.cbfmt.2f45c14f779ed0a4-cgu.14.rcgu.o" "/private/tmp/nix-build-cbfmt-0.2.0.drv-0/source/target/aarch64-apple-darwin/release/deps/cbfmt-56cc8c904f3db47c.cbfmt.2f45c14f779ed0a4-cgu.15.rcgu.o" "/private/tmp/nix-build-cbfmt-0.2.0.drv-0/source/target/aarch64-apple-darwin/release/deps/cbfmt-56cc8c904f3db47c.3k8hldo52c7uqvgc.rcgu.o" "-L" "/private/tmp/nix-build-cbfmt-0.2.0.drv-0/source/target/aarch64-apple-darwin/release/deps" "-L" "/private/tmp/nix-build-cbfmt-0.2.0.drv-0/source/target/release/deps" "-L" "/private/tmp/nix-build-cbfmt-0.2.0.drv-0/source/target/aarch64-apple-darwin/release/build/tree-sitter-3be8fea96638f87f/out" "-L" "/private/tmp/nix-build-cbfmt-0.2.0.drv-0/source/target/aarch64-apple-darwin/release/build/tree-sitter-md-5e7f50a72b71927a/out" "-L" "/private/tmp/nix-build-cbfmt-0.2.0.drv-0/source/target/aarch64-apple-darwin/release/build/tree-sitter-md-5e7f50a72b71927a/out" "-L" "/private/tmp/nix-build-cbfmt-0.2.0.drv-0/source/target/aarch64-apple-darwin/release/build/tree-sitter-md-5e7f50a72b71927a/out" "-L" "/private/tmp/nix-build-cbfmt-0.2.0.drv-0/source/target/aarch64-apple-darwin/release/build/tree-sitter-md-5e7f50a72b71927a/out" "-L" "/private/tmp/nix-build-cbfmt-0.2.0.drv-0/source/target/aarch64-apple-darwin/release/build/tree-sitter-org-1fc60d573f3ebd22/out" "-L" "/private/tmp/nix-build-cbfmt-0.2.0.drv-0/source/target/aarch64-apple-darwin/release/build/tree-sitter-org-1fc60d573f3ebd22/out" "-L" "/private/tmp/nix-build-cbfmt-0.2.0.drv-0/source/target/aarch64-apple-darwin/release/build/tree-sitter-rst-54be8877d9dc6ca1/out" "-L" "/nix/store/y24qqyfbnnxza5841hd9hnda5icmzymq-rustc-1.74.0/lib/rustlib/aarch64-apple-darwin/lib" "/private/tmp/nix-build-cbfmt-0.2.0.drv-0/source/target/aarch64-apple-darwin/release/deps/libtree_sitter_rst-75e2d2710ef3ce5d.rlib" "/private/tmp/nix-build-cbfmt-0.2.0.drv-0/source/target/aarch64-apple-darwin/release/deps/libtree_sitter_org-cf7533e91a5f00f9.rlib" "/private/tmp/nix-build-cbfmt-0.2.0.drv-0/source/target/aarch64-apple-darwin/release/deps/libtree_sitter_md-12b7a1b120c9c94d.rlib" "/private/tmp/nix-build-cbfmt-0.2.0.drv-0/source/target/aarch64-apple-darwin/release/deps/libtoml-ef4758135be1a4b9.rlib" "/private/tmp/nix-build-cbfmt-0.2.0.drv-0/source/target/aarch64-apple-darwin/release/deps/libtokio-bf866f8e346c92f7.rlib" "/private/tmp/nix-build-cbfmt-0.2.0.drv-0/source/target/aarch64-apple-darwin/release/deps/libnum_cpus-1fa28bad3be431d0.rlib" "/private/tmp/nix-build-cbfmt-0.2.0.drv-0/source/target/aarch64-apple-darwin/release/deps/libthiserror-b98a7348a03428aa.rlib" "/private/tmp/nix-build-cbfmt-0.2.0.drv-0/source/target/aarch64-apple-darwin/release/deps/libignore-33463b1af3387b7e.rlib" "/private/tmp/nix-build-cbfmt-0.2.0.drv-0/source/target/aarch64-apple-darwin/release/deps/libcrossbeam_utils-4c55fd9866aba363.rlib" "/private/tmp/nix-build-cbfmt-0.2.0.drv-0/source/target/aarch64-apple-darwin/release/deps/liblazy_static-5d4ab7ea0996e0a7.rlib" "/private/tmp/nix-build-cbfmt-0.2.0.drv-0/source/target/aarch64-apple-darwin/release/deps/libwalkdir-96ccb62ad8f45629.rlib" "/private/tmp/nix-build-cbfmt-0.2.0.drv-0/source/target/aarch64-apple-darwin/release/deps/libsame_file-9170a61c1466cc52.rlib" "/private/tmp/nix-build-cbfmt-0.2.0.drv-0/source/target/aarch64-apple-darwin/release/deps/libthread_local-d5073d14965cc882.rlib" "/private/tmp/nix-build-cbfmt-0.2.0.drv-0/source/target/aarch64-apple-darwin/release/deps/libonce_cell-34c049905450c16d.rlib" "/private/tmp/nix-build-cbfmt-0.2.0.drv-0/source/target/aarch64-apple-darwin/release/deps/libglobset-b7aff85e5e071693.rlib" "/private/tmp/nix-build-cbfmt-0.2.0.drv-0/source/target/aarch64-apple-darwin/release/deps/libfnv-cd0f88d65d6b2886.rlib" "/private/tmp/nix-build-cbfmt-0.2.0.drv-0/source/target/aarch64-apple-darwin/release/deps/liblog-e5c00d8d1e75a161.rlib" "/private/tmp/nix-build-cbfmt-0.2.0.drv-0/source/target/aarch64-apple-darwin/release/deps/libcfg_if-425cb196a193a416.rlib" "/private/tmp/nix-build-cbfmt-0.2.0.drv-0/source/target/aarch64-apple-darwin/release/deps/libbstr-41467976ab0f5963.rlib" "/private/tmp/nix-build-cbfmt-0.2.0.drv-0/source/target/aarch64-apple-darwin/release/deps/libtree_sitter-a2a4dd79cdf3f724.rlib" "/private/tmp/nix-build-cbfmt-0.2.0.drv-0/source/target/aarch64-apple-darwin/release/deps/libregex-96c08f56bab5d5c4.rlib" "/private/tmp/nix-build-cbfmt-0.2.0.drv-0/source/target/aarch64-apple-darwin/release/deps/libaho_corasick-67dce982b5db2483.rlib" "/private/tmp/nix-build-cbfmt-0.2.0.drv-0/source/target/aarch64-apple-darwin/release/deps/libregex_syntax-4cab27a7aa01e5ac.rlib" "/private/tmp/nix-build-cbfmt-0.2.0.drv-0/source/target/aarch64-apple-darwin/release/deps/libfutures-5b88b4088796e55c.rlib" "/private/tmp/nix-build-cbfmt-0.2.0.drv-0/source/target/aarch64-apple-darwin/release/deps/libfutures_executor-529cb757c3d99c45.rlib" "/private/tmp/nix-build-cbfmt-0.2.0.drv-0/source/target/aarch64-apple-darwin/release/deps/libfutures_util-8f9b044bd7d52637.rlib" "/private/tmp/nix-build-cbfmt-0.2.0.drv-0/source/target/aarch64-apple-darwin/release/deps/libmemchr-a0d220493c2a07c5.rlib" "/private/tmp/nix-build-cbfmt-0.2.0.drv-0/source/target/aarch64-apple-darwin/release/deps/libfutures_io-3e2f0f4d0cc71261.rlib" "/private/tmp/nix-build-cbfmt-0.2.0.drv-0/source/target/aarch64-apple-darwin/release/deps/libslab-e45f9bf8516ae961.rlib" "/private/tmp/nix-build-cbfmt-0.2.0.drv-0/source/target/aarch64-apple-darwin/release/deps/libfutures_channel-d6fa9ec7d7d5bb0b.rlib" "/private/tmp/nix-build-cbfmt-0.2.0.drv-0/source/target/aarch64-apple-darwin/release/deps/libpin_project_lite-2744e42f48e91026.rlib" "/private/tmp/nix-build-cbfmt-0.2.0.drv-0/source/target/aarch64-apple-darwin/release/deps/libfutures_sink-a2f7947eb4b638eb.rlib" "/private/tmp/nix-build-cbfmt-0.2.0.drv-0/source/target/aarch64-apple-darwin/release/deps/libfutures_task-e5e1d217b3b8b8ba.rlib" "/private/tmp/nix-build-cbfmt-0.2.0.drv-0/source/target/aarch64-apple-darwin/release/deps/libpin_utils-e63278e387ec30f0.rlib" "/private/tmp/nix-build-cbfmt-0.2.0.drv-0/source/target/aarch64-apple-darwin/release/deps/libfutures_core-d315d0f4b9724680.rlib" "/private/tmp/nix-build-cbfmt-0.2.0.drv-0/source/target/aarch64-apple-darwin/release/deps/libserde-8ddd13fe1ccba414.rlib" "/private/tmp/nix-build-cbfmt-0.2.0.drv-0/source/target/aarch64-apple-darwin/release/deps/libclap-f882912bc863c961.rlib" "/private/tmp/nix-build-cbfmt-0.2.0.drv-0/source/target/aarch64-apple-darwin/release/deps/libatty-974ecda4e13b08ef.rlib" "/private/tmp/nix-build-cbfmt-0.2.0.drv-0/source/target/aarch64-apple-darwin/release/deps/liblibc-ebdd3312bb2bafca.rlib" "/private/tmp/nix-build-cbfmt-0.2.0.drv-0/source/target/aarch64-apple-darwin/release/deps/libstrsim-5e0ab3b29bb0f0f8.rlib" "/private/tmp/nix-build-cbfmt-0.2.0.drv-0/source/target/aarch64-apple-darwin/release/deps/libtermcolor-22208f3512f75b9e.rlib" "/private/tmp/nix-build-cbfmt-0.2.0.drv-0/source/target/aarch64-apple-darwin/release/deps/libtextwrap-3bfc26467bf9f9d0.rlib" "/private/tmp/nix-build-cbfmt-0.2.0.drv-0/source/target/aarch64-apple-darwin/release/deps/libunicode_width-c6702804c58428ec.rlib" "/private/tmp/nix-build-cbfmt-0.2.0.drv-0/source/target/aarch64-apple-darwin/release/deps/libunicode_linebreak-41a21549014fbfb7.rlib" "/private/tmp/nix-build-cbfmt-0.2.0.drv-0/source/target/aarch64-apple-darwin/release/deps/libsmawk-083463d0cd73bb13.rlib" "/private/tmp/nix-build-cbfmt-0.2.0.drv-0/source/target/aarch64-apple-darwin/release/deps/libclap_lex-d5db9c99171050f1.rlib" "/private/tmp/nix-build-cbfmt-0.2.0.drv-0/source/target/aarch64-apple-darwin/release/deps/libos_str_bytes-addadbe71566e1d4.rlib" "/private/tmp/nix-build-cbfmt-0.2.0.drv-0/source/target/aarch64-apple-darwin/release/deps/libindexmap-4e8304f7beb60114.rlib" "/private/tmp/nix-build-cbfmt-0.2.0.drv-0/source/target/aarch64-apple-darwin/release/deps/libhashbrown-db256aad9d0ce8a4.rlib" "/private/tmp/nix-build-cbfmt-0.2.0.drv-0/source/target/aarch64-apple-darwin/release/deps/libbitflags-874464458fa759ee.rlib" "/nix/store/y24qqyfbnnxza5841hd9hnda5icmzymq-rustc-1.74.0/lib/rustlib/aarch64-apple-darwin/lib/libstd-03c53b54a0b1f97d.rlib" "/nix/store/y24qqyfbnnxza5841hd9hnda5icmzymq-rustc-1.74.0/lib/rustlib/aarch64-apple-darwin/lib/libpanic_unwind-32c6908016812ada.rlib" "/nix/store/y24qqyfbnnxza5841hd9hnda5icmzymq-rustc-1.74.0/lib/rustlib/aarch64-apple-darwin/lib/libobject-d8028f9d6e099c46.rlib" "/nix/store/y24qqyfbnnxza5841hd9hnda5icmzymq-rustc-1.74.0/lib/rustlib/aarch64-apple-darwin/lib/libmemchr-b44bd6fb2b1b839e.rlib" "/nix/store/y24qqyfbnnxza5841hd9hnda5icmzymq-rustc-1.74.0/lib/rustlib/aarch64-apple-darwin/lib/libaddr2line-c8dcf438cf6d13da.rlib" "/nix/store/y24qqyfbnnxza5841hd9hnda5icmzymq-rustc-1.74.0/lib/rustlib/aarch64-apple-darwin/lib/libgimli-703d937e20906b80.rlib" "/nix/store/y24qqyfbnnxza5841hd9hnda5icmzymq-rustc-1.74.0/lib/rustlib/aarch64-apple-darwin/lib/librustc_demangle-b7ac46c30207e45c.rlib" "/nix/store/y24qqyfbnnxza5841hd9hnda5icmzymq-rustc-1.74.0/lib/rustlib/aarch64-apple-darwin/lib/libstd_detect-4d8dae81211e4895.rlib" "/nix/store/y24qqyfbnnxza5841hd9hnda5icmzymq-rustc-1.74.0/lib/rustlib/aarch64-apple-darwin/lib/libhashbrown-d204bf1a3582275b.rlib" "/nix/store/y24qqyfbnnxza5841hd9hnda5icmzymq-rustc-1.74.0/lib/rustlib/aarch64-apple-darwin/lib/librustc_std_workspace_alloc-e447863e017a3c6b.rlib" "/nix/store/y24qqyfbnnxza5841hd9hnda5icmzymq-rustc-1.74.0/lib/rustlib/aarch64-apple-darwin/lib/libminiz_oxide-ac4d1414355d5dc4.rlib" "/nix/store/y24qqyfbnnxza5841hd9hnda5icmzymq-rustc-1.74.0/lib/rustlib/aarch64-apple-darwin/lib/libadler-6cc53510bafd9383.rlib" "/nix/store/y24qqyfbnnxza5841hd9hnda5icmzymq-rustc-1.74.0/lib/rustlib/aarch64-apple-darwin/lib/libunwind-cc9a97bec3aa68fb.rlib" "/nix/store/y24qqyfbnnxza5841hd9hnda5icmzymq-rustc-1.74.0/lib/rustlib/aarch64-apple-darwin/lib/libcfg_if-1b9937edf2e43f1a.rlib" "/nix/store/y24qqyfbnnxza5841hd9hnda5icmzymq-rustc-1.74.0/lib/rustlib/aarch64-apple-darwin/lib/liblibc-b935d087394ceac5.rlib" "/nix/store/y24qqyfbnnxza5841hd9hnda5icmzymq-rustc-1.74.0/lib/rustlib/aarch64-apple-darwin/lib/liballoc-d062ef7598770b49.rlib" "/nix/store/y24qqyfbnnxza5841hd9hnda5icmzymq-rustc-1.74.0/lib/rustlib/aarch64-apple-darwin/lib/librustc_std_workspace_core-e068a539d3cd8c41.rlib" "/nix/store/y24qqyfbnnxza5841hd9hnda5icmzymq-rustc-1.74.0/lib/rustlib/aarch64-apple-darwin/lib/libcore-c654b0e49d6357d6.rlib" "/nix/store/y24qqyfbnnxza5841hd9hnda5icmzymq-rustc-1.74.0/lib/rustlib/aarch64-apple-darwin/lib/libcompiler_builtins-738eb43ff73087cd.rlib" "-lc++" "-lc++" "-liconv" "-lSystem" "-lc" "-lm" "-L" "/nix/store/y24qqyfbnnxza5841hd9hnda5icmzymq-rustc-1.74.0/lib/rustlib/aarch64-apple-darwin/lib" "-o" "/private/tmp/nix-build-cbfmt-0.2.0.drv-0/source/target/aarch64-apple-darwin/release/deps/cbfmt-56cc8c904f3db47c" "-Wl,-dead_strip" "-nodefaultlibs" "/private/tmp/nix-build-cbfmt-0.2.0.drv-0/source/target/aarch64-apple-darwin/release/deps/cbfmt_audit_data.o" "-Wl,-u,_AUDITABLE_VERSION_INFO"[0m
[0m  [0m[0m[1m[38;5;12m= [0m[0m[1mnote[0m[0m: Undefined symbols for architecture arm64:[0m
[0m            "typeinfo for std::bad_array_new_length", referenced from:[0m
[0m                std::__throw_bad_array_new_length[abi:v160006]() in libtree_sitter_org-cf7533e91a5f00f9.rlib(scanner.o)[0m
[0m            "std::bad_array_new_length::~bad_array_new_length()", referenced from:[0m
[0m                std::__throw_bad_array_new_length[abi:v160006]() in libtree_sitter_org-cf7533e91a5f00f9.rlib(scanner.o)[0m
[0m            "___gxx_personality_v0", referenced from:[0m
[0m                std::__1::__throw_length_error[abi:v160006](char const*) in libtree_sitter_md-12b7a1b120c9c94d.rlib(scanner.o)[0m
[0m                _tree_sitter_org_external_scanner_create in libtree_sitter_org-cf7533e91a5f00f9.rlib(scanner.o)[0m
[0m            "___cxa_allocate_exception", referenced from:[0m
[0m                std::__1::__throw_length_error[abi:v160006](char const*) in libtree_sitter_md-12b7a1b120c9c94d.rlib(scanner.o)[0m
[0m                std::__throw_bad_array_new_length[abi:v160006]() in libtree_sitter_org-cf7533e91a5f00f9.rlib(scanner.o)[0m
[0m            "___cxa_free_exception", referenced from:[0m
[0m                std::__1::__throw_length_error[abi:v160006](char const*) in libtree_sitter_md-12b7a1b120c9c94d.rlib(scanner.o)[0m
[0m            "std::bad_array_new_length::bad_array_new_length()", referenced from:[0m
[0m                std::__throw_bad_array_new_length[abi:v160006]() in libtree_sitter_org-cf7533e91a5f00f9.rlib(scanner.o)[0m
[0m            "operator delete(void*)", referenced from:[0m
[0m                _tree_sitter_markdown_external_scanner_destroy in libtree_sitter_md-12b7a1b120c9c94d.rlib(scanner.o)[0m
[0m                std::__1::vector<TreeSitterMarkdown::Block, std::__1::allocator<TreeSitterMarkdown::Block>>::push_back[abi:v160006](TreeSitterMarkdown::Block&&) in libtree_sitter_md-12b7a1b120c9c94d.rlib(scanner.o)[0m
[0m                TreeSitterMarkdown::Scanner::parse_fenced_code_block(char, TSLexer*, bool const*) in libtree_sitter_md-12b7a1b120c9c94d.rlib(scanner.o)[0m
[0m                TreeSitterMarkdown::Scanner::parse_star(TSLexer*, bool const*) in libtree_sitter_md-12b7a1b120c9c94d.rlib(scanner.o)[0m
[0m                TreeSitterMarkdown::Scanner::parse_block_quote(TSLexer*, bool const*) in libtree_sitter_md-12b7a1b120c9c94d.rlib(scanner.o)[0m
[0m                TreeSitterMarkdown::Scanner::parse_list_marker_plus(TSLexer*, bool const*) in libtree_sitter_md-12b7a1b120c9c94d.rlib(scanner.o)[0m
[0m                TreeSitterMarkdown::Scanner::parse_minus(TSLexer*, bool const*) in libtree_sitter_md-12b7a1b120c9c94d.rlib(scanner.o)[0m
[0m                ...[0m
[0m            "vtable for std::length_error", referenced from:[0m
[0m                std::length_error::length_error[abi:v160006](char const*) in libtree_sitter_md-12b7a1b120c9c94d.rlib(scanner.o)[0m
[0m            NOTE: a missing vtable usually means the first non-inline virtual member function has no definition.[0m
[0m            "std::length_error::~length_error()", referenced from:[0m
[0m                std::__1::__throw_length_error[abi:v160006](char const*) in libtree_sitter_md-12b7a1b120c9c94d.rlib(scanner.o)[0m
[0m            "typeinfo for std::length_error", referenced from:[0m
[0m                std::__1::__throw_length_error[abi:v160006](char const*) in libtree_sitter_md-12b7a1b120c9c94d.rlib(scanner.o)[0m
[0m            "___cxa_throw", referenced from:[0m
[0m                std::__1::__throw_length_error[abi:v160006](char const*) in libtree_sitter_md-12b7a1b120c9c94d.rlib(scanner.o)[0m
[0m                std::__throw_bad_array_new_length[abi:v160006]() in libtree_sitter_org-cf7533e91a5f00f9.rlib(scanner.o)[0m
[0m            "operator new(unsigned long)", referenced from:[0m
[0m                _tree_sitter_markdown_external_scanner_create in libtree_sitter_md-12b7a1b120c9c94d.rlib(scanner.o)[0m
[0m                std::__1::vector<TreeSitterMarkdown::Block, std::__1::allocator<TreeSitterMarkdown::Block>>::push_back[abi:v160006](TreeSitterMarkdown::Block&&) in libtree_sitter_md-12b7a1b120c9c94d.rlib(scanner.o)[0m
[0m                TreeSitterMarkdown::Scanner::parse_fenced_code_block(char, TSLexer*, bool const*) in libtree_sitter_md-12b7a1b120c9c94d.rlib(scanner.o)[0m
[0m                TreeSitterMarkdown::Scanner::parse_star(TSLexer*, bool const*) in libtree_sitter_md-12b7a1b120c9c94d.rlib(scanner.o)[0m
[0m                TreeSitterMarkdown::Scanner::parse_block_quote(TSLexer*, bool const*) in libtree_sitter_md-12b7a1b120c9c94d.rlib(scanner.o)[0m
[0m                TreeSitterMarkdown::Scanner::parse_list_marker_plus(TSLexer*, bool const*) in libtree_sitter_md-12b7a1b120c9c94d.rlib(scanner.o)[0m
[0m                TreeSitterMarkdown::Scanner::parse_minus(TSLexer*, bool const*) in libtree_sitter_md-12b7a1b120c9c94d.rlib(scanner.o)[0m
[0m                ...[0m
[0m          ld: symbol(s) not found for architecture arm64[0m
[0m          clang-16: error: linker command failed with exit code 1 (use -v to see invocation)[0m
[0m          [0m

[1m[31merror[0m[1m:[0m could not compile `cbfmt` (bin "cbfmt") due to previous error
```

### Additional context

Add any other context about the problem here.

### Notify maintainers

@stehessel

### Metadata

Please run `nix-shell -p nix-info --run "nix-info -m"` and paste the result.

```console
[user@system:~]$ nix-shell -p nix-info --run "nix-info -m"
these 2 paths will be fetched (0.01 MiB download, 0.11 MiB unpacked):
  /nix/store/hrcw8p4cymhyb959w21ha5g1n3dm69n4-DarwinTools-1
  /nix/store/x2v35n7y8ihdwl73dzvjzjlgg4h4lsq9-nix-info
copying path '/nix/store/hrcw8p4cymhyb959w21ha5g1n3dm69n4-DarwinTools-1' from 'https://cache.nixos.org'...
copying path '/nix/store/x2v35n7y8ihdwl73dzvjzjlgg4h4lsq9-nix-info' from 'https://cache.nixos.org'...
 - system: `"aarch64-darwin"`
 - host os: `Darwin 23.2.0, macOS 14.2.1`
 - multi-user?: `yes`
 - sandbox: `no`
 - version: `nix-env (Nix) 2.15.1`
 - channels(root): `"nixpkgs"`
 - nixpkgs: `/nix/var/nix/profiles/per-user/root/channels/nixpkgs`
```

---

Add a :+1: [reaction] to [issues you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[issues you find important]: https://github.com/NixOS/nixpkgs/issues?q=is%3Aissue+is%3Aopen+sort%3Areactions-%2B1-desc

--- a/pkgs/development/tools/cbfmt/default.nix
+++ b/pkgs/development/tools/cbfmt/default.nix
@@ -1,4 +1,4 @@
-{ lib, rustPlatform, fetchFromGitHub, testers, cbfmt }:
+{ lib, rustPlatform, fetchFromGitHub, stdenv, testers, cbfmt }:
 
 rustPlatform.buildRustPackage rec {
   pname = "cbfmt";
@@ -12,6 +12,9 @@ rustPlatform.buildRustPackage rec {
   };
 
   cargoSha256 = "sha256-6oZCpjQ8t/QLFhEtF7td8KGI/kFE04pg7OELutsrJKo=";
+
+  # Work around https://github.com/NixOS/nixpkgs/issues/166205.
+  env = lib.optionalAttrs stdenv.cc.isClang { NIX_LDFLAGS = "-l${stdenv.cc.libcxx.cxxabi.libName}"; };
 
   passthru.tests.version = testers.testVersion {
     package = cbfmt;

